providers/aws/aws_provider.py

```diff
@@ -200,3 +239,37 @@ def assume_role(audit_info: AWS_Audit_Info) -> dict:
 
     else:
         return assumed_credentials
+
+
+def get_organizations_metadata(assumed_credentials):
+    try:
+        organizations_client = client(
+            "organizations",
+            aws_access_key_id=assumed_credentials["Credentials"]["AccessKeyId"],
+            aws_secret_access_key=assumed_credentials["Credentials"]["SecretAccessKey"],
+            aws_session_token=assumed_credentials["Credentials"]["SessionToken"],
+        )
+        organizations_metadata = organizations_client.describe_account(
+            AccountId=current_audit_info.audited_account
+        )
+        list_tags_for_resource = organizations_client.list_tags_for_resource(
+            ResourceId=current_audit_info.audited_account
+        )
+        # Convert Tags dictionary to String
+        account_details_tags = ""
+        for tag in list_tags_for_resource["Tags"]:
+            account_details_tags = (
+                account_details_tags + tag["Key"] + ":" + tag["Value"] + ","
+            )
```

Comment:
You can do the following to concatenate strings:
```suggestion
            account_details_tags +=  tag["Key"] + ":" + tag["Value"] + ","
```